source/code/scxsystemlib/disk/statisticallogicaldiskenumeration.cpp

```diff
@@ -217,6 +217,10 @@ namespace SCXSystemLib
         scxulong total_rPercent = 0;
         scxulong total_wPercent = 0;
         scxulong total_tPercent = 0;
+
+	set<wstring> diskSet;
+	pair<set<wstring>::iterator,bool> Pair;
```

Comment:
These are indented with tabs rather than spaces like the rest of the file, causing indentation issues. Please fix.